atomic_reactor/plugins/cachi2_postprocess.py

```diff
@@ -278,9 +279,16 @@ def inject_into_build_dir(
             self.log.debug(
                 "copy method used for cachi2 build_dir_injecting: %s", copy_method.__name__)
 
-            copytree(
-                remote_source.sources_path, dest_dir,
-                symlinks=True, copy_function=copy_method, dirs_exist_ok=True)
+            for path in copy_paths:
+                try:
+                    copytree(
+                        remote_source.sources_path/path, dest_dir/path,
+                        symlinks=True, copy_function=copy_method, dirs_exist_ok=True)
+                except FileNotFoundError:
```

Comment:
can we rather check `if path.exist()` in advance ?

Reply:
Fixed.

Thanks! It's *much* better now